.travis.yml

```diff
@@ -3,18 +3,16 @@ node_js:
   - "8"
   - "6"
   - "4"
-  - "0.12"
-  - "0.10"
-before_install:
-  - 'if [ "${TRAVIS_NODE_VERSION}" = "0.6" ]; then npm install -g npm@1.3 ; elif [ "${TRAVIS_NODE_VERSION}" != "0.9" ]; then case "$(npm --version)" in 1.*) npm install -g npm@1.4.28 ;; 2.*) npm install -g npm@2 ;; esac ; fi'
```

Comment:
these before_install lines need to be preserved, even if we don't currently test on those node versions.